tools/isobuild/meteor-npm.js

```diff
@@ -14,6 +14,7 @@ var buildmessage = require('../utils/buildmessage.js');
 var utils = require('../utils/utils.js');
 var runLog = require('../runners/run-log.js');
 var Profile = require('../tool-env/profile.js').Profile;
+import { version as npmVersion } from 'npm';
```

Comment:
This can be `export { version as npmVersion } from "npm"`, and then you don't need to export it below!